style.go

```diff
@@ -264,6 +264,7 @@ func (s Style) Render(str string) string {
 
 	// Strip newlines in single line mode
 	if inline {
+		str = strings.Trim(str, "\n")
```

Comment:
Just a heads up: this also trims leading newlines. It may or may not be what we want here, I'll still have to think about the implications some more.